iosApp/iosAppTests/Support/PublicAPI/Conversations.swift

```diff
@@ -14,14 +14,24 @@ extension ApiHelper {
         let agentId = TestConfig.shared.config?.agentId ?? ""
         changePresence(presenceName: "On Queue", userID: agentId)
         guard let conversation = waitForConversation() else {
-            XCTFail("Failed to receive a new conversation to answer.")
+            print("Failed to receive a new conversation to answer.")
             return nil
         }
         sendConnectOrDisconnect(conversationInfo: conversation, connecting: true, wrapup: false)
         changePresence(presenceName: "Available", userID: agentId)
         return getConversation(conversationId: conversation.conversationId)
     }
 
+    public func disconnectExistingConversations() {
+        guard let conversationList = getSmsConversations()?.value(forKey: "entities") as? [JsonDictionary] else {
+            XCTFail("Failed to get the existing conversation list.")
+            return
+        }
+        for conversation in conversationList {
+            sendConnectOrDisconnect(conversationInfo: ConversationInfo(json: conversation), connecting: false)
+        }
+    }
+
     private func getSmsConversations() -> JsonDictionary? {
```

Comment:
What does the "Sms" stand for?

Reply:
Back when this ACD type was first released, it was the "sms" media type. Now it's called the "Message" media type but the function name I use for it hasn't been updated with that just yet.